views/signup.jade

```diff
@@ -66,9 +73,12 @@ block content
         - var useLocal = glob.auth.local && glob.auth.local.useLocal
         - var useSocial = useAdfs || useGithub || useGoogle || useOauth2
 
-        h4.
-            By signing up to this portal, you agree to our <a href='/content/terms-and-conditions'>Terms
-            and Conditions</a> and the <a href='/content/privacy-policy'>Privacy/Cookie Policy</a>.
+        h4
+            | By signing up to this portal, you agree to our&nbsp;
+            a(href="/content/terms-and-conditions" title="Terms and Conditions") Terms and Conditions
+            | &nbsp;and the&nbsp;
+            a(href="https://clarivate.com/legal/privacy-statement/" title="Privacy/Cookie Policy") Privacy/Cookie Policy
```

Comment:
This is open source code we cannot hard code https://clarivate.com/legal/privacy-statement/

Reply:
This should be brought up to Ellen to decide the best way to handle this. We are hosting this on our end right?

Reply:
Anything we add to "wicked.portal" repo needs to go into open source code.